Exiled.API/Features/Roles/Scp3114Role.cs

```diff
@@ -204,5 +204,14 @@ public void ResetIdentity()
         /// <param name="alreadySpawned">The List of Roles already spawned.</param>
         /// <returns>The Spawn Chance.</returns>
         public float GetSpawnChance(List<RoleTypeId> alreadySpawned) => 0;
+
+        /// <summary>
+        /// Gets or sets the warning time seconds
+        /// </summary>
+        public float WarningTime
+        {
+            get => Identity._warningTimeSeconds;
+            set => Identity._warningTimeSeconds = value;
```

Comment:
Warning is probably not configurable i just need you to verify than it's actually work

Reply:
Much like how DisguiseDuration functions. WarningTime also works in a similar manner.

I have tested it and I have verified that it is functional